components/tally/actions/get-responses/get-responses.mjs

```diff
@@ -0,0 +1,38 @@
+import tally from "../../tally.app.mjs";
+
+export default {
+  name: "Get Responses",
+  version: "0.0.1",
+  key: "tally-get-responses",
+  description: "Get a list of responses. [See docs here](https://tallyso.notion.site/Tally-OAuth-2-reference-d0442c679a464664823628f675f43454)",
+  type: "action",
+  dedupe: "unique",
+  props: {
+    tally,
+    formId: {
+      propDefinition: [
+        tally,
+        "forms",
+      ],
+    },
+    page: {
+      label: "Page",
+      description: "The page for retrieve responses, starts with 1",
+      type: "integer",
+      min: 1,
+      default: 1,
+      optional: true,
+    },
```

Comment:
I think pagination should be handled automatically, without requiring a user input.
For example, remove this prop and handle pagination inside `tally.getResponses()` to receive all responses